lib/node_modules/@stdlib/lapack/base/dlacn2/test/test.dlacn2.js

```diff
@@ -0,0 +1,222 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2025 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+/* eslint-disable array-element-newline */
+
+// MODULES //
+
+var tape = require( 'tape' );
+var Float64Array = require( '@stdlib/array/float64' );
+var Int32Array = require( '@stdlib/array/int32' );
+var dgemv = require( '@stdlib/blas/base/dgemv' );
+var dcopy = require( '@stdlib/blas/base/dcopy' );
+var dlacn2 = require( './../lib/dlacn2.js' );
+
+
+// TESTS //
+
+tape( 'main export is a function', function test( t ) {
+	t.ok( true, __filename );
+	t.strictEqual( typeof dlacn2, 'function', 'main export is a function' );
+	t.end();
+});
+
+tape( 'the function has an arity of 7', function test( t ) {
+	t.strictEqual( dlacn2.length, 7, 'returns expected value' );
+	t.end();
+});
+
+tape( 'the function returns expected values when calculating the one-norm of a square matrix for N = 1', function test( t ) {
+	var expectedISAVE;
+	var expectedKASE;
+	var expectedISGN;
+	var expectedEST;
+	var expectedV;
+	var ISAVE;
+	var KASE;
+	var ISGN;
+	var work;
+	var EST;
+	var X;
+	var V;
+	var A;
+
+	A = new Float64Array( [ -25.0 ] );
+
+	KASE = new Int32Array( 1 );
+	EST = new Float64Array( 1 );
+	ISGN = new Int32Array( 1 );
+	ISAVE = new Int32Array( 3 );
+	X = new Float64Array( 1 );
+	V = new Float64Array( 1 );
+
+	work = new Float64Array( 1 );
+
+	while ( true ) {
```

Comment:
May be good to add a comment before the `while` loop explaining, conceptually, what you are doing here.